rotkehlchen/tests/db/test_db_upgrades.py

```diff
@@ -1339,3 +1342,46 @@ def test_old_versions_raise_error(user_data_dir):  # pylint: disable=unused-argu
             msg_aggregator=msg_aggregator,
         )
     assert 'Your account was last opened by a very old version of rotki' in str(upgrade_exception)
+
+
+def test_ongoing_upgrades(user_data_dir):
+    _use_prepared_db(user_data_dir, 'v33_rotkehlchen.db')
+    msg_aggregator = MessagesAggregator()
+    db = _init_db_with_target_version(
+        target_version=33,
+        user_data_dir=user_data_dir,
+        msg_aggregator=msg_aggregator,
+    )
+    with db.user_write() as write_cursor:
+        db.set_setting(  # Pretend that an upgrade was started
+            write_cursor=write_cursor,
+            name='ongoing_upgrade_from_version',
+            value=33,
+        )
+    db.logout()
+    # There are no backups, so it is supposed to raise an error
+    with pytest.raises(DBUpgradeError):
+        _init_db_with_target_version(
+            target_version=34,
+            user_data_dir=user_data_dir,
+            msg_aggregator=msg_aggregator,
+        )
+
+    # Add a backup
+    shutil.copy(
+        os.path.join(
+            os.path.dirname(os.path.dirname(os.path.realpath(__file__))),
+            'data',
+            'v33_rotkehlchen.db',
+        ),
+        os.path.join(user_data_dir / f'{ts_now()}_rotkehlchen_db_v33.backup'),
```

Comment:
funny thing is you are actually using the pathlib module here :D. The `/` operator does this. And then you turn it back to a string to push to os.path.join
```suggestion
        user_data_dir / f'{ts_now()}_rotkehlchen_db_v33.backup',
```